packages/p/polylith/package.yml

```diff
@@ -0,0 +1,23 @@
+name       : polylith
+version    : 0.2.18
+homepage   : https://polylith.gitbook.io/polylith/
+release    : 1
+source     :
+  - https://github.com/polyfy/polylith/archive/refs/tags/v0.2.18.tar.gz: 213e75ac3933a27ce8fc2a509904e8faa29ee8ec6fd914e824eb68ca2f99c965
+license    : EPL-1.0
+component  : programming.tools
+networking : yes
+summary    : Clojure tool for managing polylith projects
+description: |
+  The poly tool helps you manage your polylith projects which introduces the architectural concept of “service level building blocks” which can be combined like LEGO® bricks to build our services and systems.
+environment: |
+  export JAVA_CMD=/usr/lib64/openjdk-17/bin/java
+builddeps  :
+  - clojure-tools
+rundeps    :
+  - openjdk-17
+build      : |
+  clojure -T:build install :project poly
+install    : |
+  install -Dm00755 $pkgfiles/poly $installdir/usr/local/bin/poly
```

Comment:
This should be going to `/usr/bin`. `/usr/local` is for programs that aren't managed by the system's package manager.

Reply:
okay thanks for spotting that updated